internal/gapicgen/git/github.go

```diff
@@ -268,7 +268,7 @@ git push origin $BRANCH_NAME
 		Title: &t,
 		Body:  &body,
 		Head:  github.String(fmt.Sprintf("googleapis:" + gocloudBranchName)),
-		Base:  github.String("master"),
+		Base:  github.String(""),
```

Comment:
main?

Reply:
I am not sure what happened here, good catch!